docs/src/content/docs/fr/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Page Introuvable
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, nous avons un problème.</strong> Nous n’avons pas pu trouver cette page.<br>Vérifiez l’URL ou essayez la barre de recherche.
+  actions:
+    - text: Retourner à l’accueil
+      icon: right-arrow
+      link: /
```

Comment:
Thanks for this contribution @cbontems!

Looks great. Here we can link to the French homepage though:
```suggestion
      link: /fr/
```